packages/app/src/pages/WelcomePage/WalletPage.test.tsx

```diff
@@ -0,0 +1,55 @@
+import "cross-fetch/polyfill";
+import {
+  fireEvent,
+  renderWithRouter,
+  screen,
+  waitFor,
+} from "@fuels-ui/test-utils";
+
+import App from "~/App";
+
+export const checkFirstLoad = async () => {
+  await renderWithRouter(<App />, { route: "/pool/list" });
+
+  const createWalletBtn = await screen.findByTestId("createWalletBtn");
+  expect(createWalletBtn).toBeInTheDocument();
+};
+
+export const createWallet = async () => {
```

Comment:
I think you're making wrong tests here, for `WalletPage` you need to test things related to the wallet widget in the header... if you want to test creating wallet, add funds and all welcome steps, you need to create these tests in the `Welcome` system

Reply:
renamed test to Welcome